examples/react-design-system/src/components/ReviewsSlider/ReviewsSlider.builder.js

```diff
@@ -45,11 +45,17 @@ export const ReviewsSliderBuilderConfig = {
       ],
       defaultValue: [
         {
-          reviewAuthor: 'Jane Doe',
+          reviewAuthor: 'Engineering',
           image:
             'https://cdn.builder.io/api/v1/image/assets%2Fpwgjf0RoYWbdnJSbpBAjXNRMe9F2%2Ffb27a7c790324294af8be1c35fe30f4d',
           reviewText: '"You guys are the best"',
         },
+        {
+          reviewAuthor: 'Marekting',
```

Comment:
small typo - `Marketing`

Reply:
might be also good to make this component auto rotate since it's really hard to see the dots below it to make it clear it's a carousel